src/components/oscal-utils/OSCALCatalogUtils.js

```diff
@@ -0,0 +1,19 @@
+import matchingProp from "./OSCALPropUtils";
+
+/**
+ * Determines if a control is withdrawn.
+ *
+ * TODO: This is probably 800-53 specific and it should be made more
+ * generic to allow the library to work with other frameworks.
+ * https://github.com/EasyDynamics/oscal-react-library/issues/504
```

Comment:
This is now handled. `props[@name="status" and @value="withdrawn"]` is how to do this per the spec and we now handle the namespace field correctly for this.